src/i18n.js

```diff
@@ -0,0 +1,39 @@
+import i18n from 'i18n-js';
+// import * as RNLocalize from ‘react-native-localize’;
+// import memoize from ‘lodash.memoize’; // Use for caching/memoize for better performance
+import en from '../languageBase/en.json';
```

Comment:
replace `en` with 'English` as you are you full name convention for all other languages